src/modules/claims/claims.service.ts

```diff
@@ -216,6 +218,38 @@ export class ClaimsService {
         return this._cacheClient.issueClaim(this._signerService.did, message);
     }
 
+    /**
+     * @description Registers issued onchain claim with Claim manager
+     *
+     * @param claimId - id of signed onchain claim
+     */
+    async registerOnchain(claimId: Claim["id"]) {
+        const claim = await this.getClaimById(claimId);
```

Comment:
I think it is better if it is the `Claim` here so that we don't force another call to the cache-server
```suggestion
    async registerOnchain(claim: Claim) {
```